pdp/static/js/pdp_auth.js

```diff
@@ -269,4 +321,21 @@ window.pdp = (function (my, $) {
     };
 
     return my;
-}(window.pdp, jQuery));
+}(window.pdp, jQuery, hello));
+
+var GOOGLE_CLIENT_ID = {
```

Comment:
Public key is required in the authorization request. The Oauth provider is configured with a set redirect uri (per app/host) that will only send the auth token back to the configured endpoint. We do not use explicit grant/auth code grant which would require secret keys (projected) and server-server authorization.